### ISSUE TITLE
Impaired renal tubular reabsorption of magnesium 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6862]
**Preferred term label:**
Impaired renal tubular reabsorption of magnesium 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of magnesium.
PMID: 25287933

**Comment**
96 % of filtered magnesium is reabsorbed in the tubule. 10% –30% of the filtered magnesium is absorbed in the proximal tubule, the thick ascending limb absorbs 40%–70% of filtered magnesium.

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bartter-syndrome, FHHNC (# 248190), Gitelman’s-syndrome (# 263800), RCAD-syndrome (#137920) 

**Associated Genes**
CLDN16, CLDN19, HNF1B, TRPM6, NKCC2, ROMK, ClC-Kb, CaSR, FXYD2, CNNM2, Kv1.1, Kir4.1, NCCCLDN16

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
This was HP_0033759. I am updating the definition as above, thanks
